src/dune_util/log.ml

```diff
@@ -8,11 +8,7 @@ module File = struct
     | Out_channel of out_channel
 end
 
-type real =
-  { oc : out_channel option
-  ; buf : Buffer.t
-  ; ppf : Format.formatter
-  }
+type real = { oc : out_channel option }
```

Comment:
Why keep this as a record?

Reply:
Finite amount of time. I can't refactor absolutely everything unfortunately.